flake.nix

```diff
@@ -125,9 +125,15 @@
               [ ]);
 
           LLVM_SYS_130_PREFIX = "${llvmPkgs.llvm.dev}";
+
           # nix does not store libs in /usr/lib or /lib
+          # for libgcc_s.so.1
+          NIX_LIBGCC_S_PATH =
+            if pkgs.stdenv.isLinux then "${pkgs.stdenv.cc.cc.lib}/lib" else "";
```

Comment:
`libgcc_s.so.1` used to be present in the NIX_GLIBC_PATH but this is no longer the case with recent releases.